roles/percona_xtradb_cluster/vars/main.yml

```diff
@@ -27,12 +27,11 @@ _percona_xtradb_cluster_spec:
         ports:
           - name: metrics
             containerPort: 9104
-        livenessProbe:
-          httpGet:
-            port: metrics
-            path: /
```

Comment:
Does it make sense to switch this to a `readinessProbe` perhaps?

Reply:
let me test it once again

Reply:
Yes. Changed to `readinessProbe`.